src/components/SocialMediaLinks/SocialMediaLinks.tsx

```diff
@@ -0,0 +1,169 @@
+import {
+    chakra,
+ //   useStyleConfig,
+} from "@chakra-ui/react";
+import List from "../List/List";
+import Link from '../Link/Link';
+import Icon from "../Icons/Icon";
+import React, { forwardRef } from "react";
+
+export const socialMediaLinkTypeArray = [
+    "blogs",
+    "facebook",
+    "instagram",
+    "pinterest",
+    "soundcloud",
+    "tiktok",
+    "tumbler",
+    "twitter",
+    "youtube",
+] as const;
+export type SocialMediaLinkType = typeof socialMediaLinkTypeArray[number];
+
+export const colorTypeArray = [
+    "link",
+    "textDefault",
+    "textInverse",
+] as const;
+export type ColorType = typeof colorTypeArray[number];
+
+export const borderTypeArray = [
+    "none",
+    "circular",
+    "straight",
+] as const;
+export type BorderType = typeof borderTypeArray[number];
+
+export const layoutTypeArray = [
+    "column",
+    "row",
+] as const;
+export type LayoutType = typeof layoutTypeArray[number];
```

Comment:
Yes, this is correct! This `LayoutType` type does already exist and if you can reuse it by importing it, that'd be great: https://github.com/NYPL/nypl-design-system/blob/development/src/helpers/types.ts